galaxyproject/galaxy
# Bugfix/8488/rule builder fails for one bad url

URLs uploaded now use Data Fetch. 
Fix for #8488

To test: 
1. Try to upload these URLs using the Default or the Collection upload modal:
https://zenodo.org/api/files/55133323-b15b-45b4-98c9-dda00288b53f/nurse.bw
https://zenodo.org/api/files/55133323-b15b-45b4-98c9-dda00288b53f/nurse_Amel4.5_accepted_hits.bam

2.  On main/usegalaxy, they will both fail, because one URL is bad. On this branch, only the bad branch will fail. 
3. On Composite, the job still uses the upload1 tool, because if a URL fails, the dataset should be in an error state.
